src/transformation/utils/export.ts

```diff
@@ -98,9 +98,11 @@ export function isSymbolExportedFromScope(
         return false;
     }
 
-    // ts.Iterator is not a ES6-compatible iterator, because TypeScript targets ES5
-    const it: Iterable<ts.Symbol> = { [Symbol.iterator]: () => scopeSymbol!.exports!.values() };
-    return [...it].includes(symbol);
+    if (symbol.declarations.find(d => findFirstNodeAbove(d, ts.isBlock))) {
```

Comment:
Having a block above isn't enough to match symbol identities there

```ts
export let a = 1;
(() => {
    a = 2;
})();
```

```ts
export let a = 1;
switch (0 as any) {
    case 1:
        let a = 2;
    case 2:
        a = 3
}
```

Reply:
No longer using `ts.isBlock`

But I am curious what is expected to happen in a switch statement like this